test/integration/query-interface/schemas.test.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import { afterEach } from 'mocha';
+import { spy } from 'sinon';
+import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
+import { sequelize } from '../support';
+
+const queryInterface = sequelize.queryInterface;
+
+const testSchema = 'testSchema';
+
+if (sequelize.dialect.supports.schemas) {
+  describe('QueryInterface#{create}Schema', async () => {
+
+    // Clear the test schema if it's been added
+    afterEach(async () => {
```

Comment:
This is already what the global afterEach does (unless `disableDatabaseResetForSuite` is called in the suite), so you don't need to have this :)